rand-core/src/lib.rs

```diff
@@ -81,6 +81,14 @@ pub mod le;
 /// in this trait directly, then use the helper functions from the [`impls`]
 /// module to implement the other methods.
 /// 
+/// It is recommended that implementations also implement:
+/// 
+/// - `Debug` but such that no internal details are output
```

Comment:
For CSPRNGs this seems like something to recommend to me, but I am starting to feel like it matters less for small rngs. There it may even be desirable to print the state.

Reply:
I doubt it matters much; the most that most people would want to do with the state is check whether it matches another state printed in the log, and it's not a very good way of checking states are equal in the first place (serialisation would be a better option).